irctest/controllers/ergo.py

```diff
@@ -58,6 +58,11 @@
             "enabled": True,
             "method": "strict",
         },
+        "login-throttling": {
+            "enabled": True,
+            "duration": "1m",
+            "max-attempts": 3,
+        },
```

Comment:
what does that do?

Reply:
It has no effect on these tests, but it's part of the default/recommended configuration so I wanted the integration tests to exercise it